src/gui/userstatus.cpp

```diff
@@ -30,16 +30,37 @@ Q_LOGGING_CATEGORY(lcUserStatus, "nextcloud.gui.userstatus", QtInfoMsg)
 
 UserStatus::UserStatus(QObject *parent)
     : QObject(parent)
-    , _message("")
 {
 }
 
 UserStatus::Status UserStatus::stringToEnum(const QString &status) const 
 {
+    // it needs to match the Status enum
+    const QHash<QString, Status> preDefinedStatus{{"online", Online},
```

Comment:
Maybe there was a reason for making that a local variable that I overlooked, but why is not a static local variable?

Reply:
It's kind of a micro-optimization really, but adding static can't hurt here really.

Reply:
See Kevin's comments in the previous PR #2505.